src/tools/fuzz_checker/fuzzer.cpp

```diff
@@ -12,8 +12,12 @@
 #include <vector>
 int MAIN(int argc, char **argv);
 
-void EXIT(int status) {
-    throw status;
+struct Exit {
+    int status;
+};
+
+[[noreturn]] void EXIT(int status) {
+    throw Exit{status};
```

Comment:
My only gripe about this PR is the spacing of these braces. I'm not that used to C++, have you used clang-format (with who knows which config)?

Reply:
Out of 7 default configs for clang-format, four of them use `Exit{status}`, 2 use `Exit{ status }` and 1 uses `Exit { status }`.
Which do you prefer? xD

Reply:
Lol! The one which is more in agreement with the rest of the file. If that too much effort no biggie, I'll merge this as it is.

Reply:
The current version is consistent because this syntax is not used anywhere else.